src/client/src/orangehrmAdminPlugin/pages/languagePackage/language-package.scss

```
@@ -27,20 +29,16 @@
 }
 
 .oxd-table-cell-actions{
```

```suggestion
.oxd-table-cell-actions {
```
happy to see formatted code